src/routes/auth.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { AuthController } from '../controllers/AuthController';
+
+const router = express.Router();
+const authController = new AuthController();
+
```

Comment:
Can we add a route that takes in the token and returns whether the token is valid and their username